plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineProjectWizard.java

```diff
@@ -98,7 +98,9 @@ public boolean performFinish() {
       config.setEclipseProjectLocationUri(page.getLocationURI());
     }
 
-    config.setAppEngineLibraries(page.getSelectedLibraries());
+    if (page.isLibrarySelectorGroupAvailable()) {
+      config.setAppEngineLibraries(page.getSelectedLibraries());
+    }
```

Comment:
You might not need this change, if you make `getSelectedLibraries()` work properly under all circumstances, i.e., make it return an empty list. Then, you can remove `isLibrarySelectorGroupAvailable()` too.